gtfs-realtime/spec/en/trip-modifications.md

```diff
@@ -0,0 +1,64 @@
+# Trip Modifications
+
+A `TripModifications` message identifies a list of similar `trip_ids` from the (CSV) GTFS sharing the same stop pattern, which are all affected by a particular modification, such as a detour.
+
+## SLO: Service-level objective
+
+The frequency of data updates is expected to be approximately hourly (~24 times/day). Ingestion time may depend on the total number of affected trips. Consumers are expected to ingest a single TripModification within 5 minutes, and a feed with hundreds of detours within 20 minutes.
+
+## TripModifications
+
+The modification is in effect on all of the listed service\_dates, until it is removed from the feed. On any given service date, a trip MUST NOT be assigned to more than one `TripModifications` object.
+
+There MAY be multiple `TripModifications` for a given stop pattern. It may be desirable to split the trips into multiple modifications e.g. if the `propagated_modification_delay` changes significantly, over the course of the detour.
```

Comment:
question: these two sentences appear to contradict each other.
- if it's not possible for a trip to have multiple `TripModification`s applied to it, is this saying that a single trip can only have one detour?
- if it is possible for a trip to have multiple `TripModification`s applied to it, how is the modified ID generated?

Additionally, https://github.com/google/transit/pull/403/files#diff-a1a14af3b4aed7d10a3b167231b52fdb483cf71b88f1655ed2e28bccd18800ceR1074 refers to multiple `TripModification`s applying to a single trip, so either this or that should be updated.

Reply:
There can be multiple `TripModifications` for the same stop pattern, but only one `TripModifications` per trip per service date. As mentioned, the delays values can potentially be different enough to warrant different `TripModifications` if needed. Or you could have two different detours for the same route with the same stop patter during one day (for example, a rush hour detour and a detour outside of rush hour). 

I think there might be a confusion between `TripModifications` and `Modification`. A `TripModifications` applies on a list of trip and [can contains multiple](https://github.com/google/transit/pull/403/files/9a0d6d5e9c790c2f22a88809a2993a9c2c14077d#diff-a1a14af3b4aed7d10a3b167231b52fdb483cf71b88f1655ed2e28bccd18800ceR1102) `Modification`. A `Modification` applies on a range of stop in the selected trips. The names are indeed very similar, which might be confusing.

Reply:
Ah yes, that is a bit confusing, but it does make a lot of my questions go away. I think there are some other copy edits which might make it clearer that I'll go through and add.